corporate/views/support.py

```diff
@@ -464,13 +465,24 @@ def get_realm_admin_emails_as_string(realm: Realm) -> str:
 
 
 def get_remote_servers_for_support(
-    email_to_search: Optional[str], hostname_to_search: Optional[str]
+    email_to_search: Optional[str], uuid_to_search: Optional[str], hostname_to_search: Optional[str]
 ) -> List["RemoteZulipServer"]:
     remote_servers_query = RemoteZulipServer.objects.order_by("id").exclude(deactivated=True)
 
     if email_to_search:
         return list(remote_servers_query.filter(contact_email__iexact=email_to_search))
 
+    if uuid_to_search:
+        remote_servers_set = set(remote_servers_query.filter(uuid__iexact=uuid_to_search))
+        remote_realm_matches = (
```

Comment:
This should only match one object, either a remote server or remote realm, so maybe we only check for remote realm UUIDs if there is no server match?

Reply:
Probably doesn't super matter whether we also check the other set or not.